packages/2d/src/components/Layout.ts

```diff
@@ -157,18 +158,32 @@ export class Layout extends Node {
   @initial(null)
   @signal()
   public declare readonly fontFamily: SimpleSignal<string | null, this>;
-  @initial(null)
   @signal()
-  public declare readonly fontSize: SimpleSignal<number | null, this>;
+  public declare readonly fontSize: SimpleSignal<number, this>;
+
+  protected computeFontSize(): number {
+    this.requestLayoutUpdate();
```

Comment:
This is a problem btw. I've noticed this when testing. Technically, `this.requestFontUpdate()` should be enough to update all important font values but for some reason it doesn't work. `this.requestLayoutUpdate()` is a bit more expensive so it would be nice to get the former to work.

Reply:
I abstracted these functions into `@computeStyle`, which always uses this.requestLayoutUpdate(). Do you think the performance gains here warrant the creation of a `@computeFontStyle` too?